### PR TITLE
Add support to optionally change the actions text.

### LINKED ISSUE #87: i18n support of actions
The actions text are hardcoded in English. It would be useful to have the text localized or provide a mechanism for providing localized strings.

### PR DESCRIPTION
**Pull request by GCheung55**
Fixes #87.

### CODE REVIEW

File: src/embed.ts [hunk 1]
```diff
@@ -162,6 +179,9 @@ export default async function embed(
           { export: { svg: true, png: true }, source: true, compiled: false, editor: true },
           opt.actions || {}
         );
+  const i18n = mergeDeep<I18n>({}, I18N, opt.i18n || {});
```
Comment (domoritz) [inline src/embed.ts]: No need to do a deep merge here. Just use an object spread.

File: test/embed.test.ts [hunk 2]
```diff
@@ -93,3 +93,22 @@ test('guessMode from Vega-Lite spec', () => {
...
+  const { COMPILED_ACTION, EDITOR_ACTION, PNG_ACTION, SOURCE_ACTION, SVG_ACTION } = i18n;
+  const el = document.createElement('div');
+  let ctrl;
+  let ctrlChildren;
+  await embed(el, vlSpec, {actions: true, i18n});
+
+  ctrl = el.children[2].children[0];
+  ctrlChildren = ctrl.children;
+
+  expect(ctrl.childElementCount).toBe(5);
```
Comment (domoritz) [inline test/embed.test.ts]: In the test, please use the string and not a variable .

File: README.md [hunk 3]
```diff
@@ -116,7 +116,14 @@ var opt = {
 
   "hover" : ...,
 
-  "runAsync": ...
+  "runAsync": ...,
```
Comment (domoritz) [inline README.md]: Maybe add an enpty line

File: README.md [hunk 4]
```diff
@@ -141,6 +148,7 @@ var opt = {
 | `sourceFooter` | String   | HTML to inject into the end of the page generated by the "View Source" and "View Vega" action link. The text will be added immediately before the closing `` tag. |
 | `hover`       | Boolean  | Enable [hover event processing](https://vega.github.io/vega/docs/api/view/#view_hover). |
 | `runAsync`     | Boolean  | Use [`runAsync`](https://vega.github.io/vega/docs/api/view/#view_runAsync) instead of [`run`](https://vega.github.io/vega/docs/api/view/#view_run). |
+| `i18n`         | Object   | This property can take a key-value mapping object that maps keys (`COMPILED_ACTION`, `EDITOR_ACTION`, `PNG_ACTION`, `SOURCE_ACTION`, `SVG_ACTION`) to string values for the action's text. By default, the text is in English. |
```
Comment (domoritz) [inline README.md]: Key value mapping object that maps is redundant

File: test/embed.test.ts (hunk 2 above)
Comment (GCheung55) [reply test/embed.test.ts]: Changes made. Please check.

File: README.md (hunk 3 above)
Comment (GCheung55) [reply README.md]: Changes made. Please check.

File: src/embed.ts [hunk 5]
```diff
@@ -46,6 +54,7 @@ export interface EmbedOptions {
   editorUrl?: string;
   hover?: boolean;
   runAsync?: boolean;
+  i18n?: I18n;
```
Comment (domoritz) [inline src/embed.ts]: Replace with `Partial<typeof I18N>`.

File: src/embed.ts [hunk 6]
```diff
@@ -26,6 +26,14 @@ export interface Actions {
   editor?: boolean;
 }
 
+export interface I18n {
```
Comment (domoritz) [inline src/embed.ts]: Unused variable?

**Pull request merged**